historical_system_profiles/archiver.py

```diff
@@ -143,9 +146,10 @@ def _check_and_send_notifications(
                 # Add each baseline to our event, then send notification only once
                 # containing the whole list of events for this system
                 drift_found = True
-                # currently using baseline_id in place of baseline_name for testing
-                # will need to make another call to api to get this later
-                event.add_drifted_baseline(baseline_id, baseline_id, comparison)
+                baseline_name = fetch_baselines([baseline_id], service_auth_key, logger)[0][
+                    display_name
+                ]
```

Comment:
My guess is that it should be like this:

```
baseline_name = fetch_baselines([baseline_id], service_auth_key, logger)[0][
                    'display_name'
                ]
```

So we get the value from the dict and not the variable in the method parameter (which is the system/host display_name)

Reply:
Good catch, thanks!